examples/llama_stack_client_examples/README.fireworks.md

```diff
@@ -0,0 +1,9 @@
+##
+
+https://github.com/meta-llama/llama-stack/blob/main/distributions/fireworks/run.yaml
+
+##
+
+```sh
+docker-compose -f fireworks-server-config.yaml up
```

Comment:
I think this was maybe meant to be `docker-compose -f fireworks-server-compose.yaml`
```suggestion
docker-compose -f fireworks-server-config.yaml up
```